syntax = "proto3";

package GrpcAgent;

service Publisher {
  rpc PublishMessage (PublisherRequest) returns (PublisherReply);
}

message PublisherRequest {
  string topic = 1;
  string content = 2;
}

message PublisherReply {
  bool isSuccess = 1;
}
